### DISCUSSION TITLE
Vercel Build Error: Module not found: Can't resolve '/vercel/path0/next/node_modules/@headlessui/react/dist/components/transition/transition'

### DISCUSSION THREAD

**Discussion by joselvelez** [vercel/next.js#54249] [Help]
### Summary

I just started getting this build error:

Module not found: Can't resolve '/vercel/path0/next/node_modules/@headlessui/react/dist/components/transition/transition'
17:06:43.261

Any clues how to resolve this? it works fine on localhost.

**Comment by heijmerikx** [comment]
It could be that you've deployed a newer version of @headlessui/react, deployed it, and later downgraded the package again and redeployed it. At least, to me that might be the cause of [this potential case](https://vercel.com/guides/how-do-i-resolve-a-module-not-found-error). Since between version i.e. 1.7.17 and 2.1.0 the path for the transition component is different.

~~I've not yet found a solution to fix it in Vercel yet, but if I do, I'll post an update.~~

I ended up upgrading to the new v2 version...
